app/src/debug/res/values/strings.xml

```diff
@@ -1,3 +1,3 @@
 <resources>
-    <string name="app_name">Contest Debug</string>
+    <string name="app_name">Contest App</string>
```

Comment:
It may be helpful to have both. If you download the build from the OTA server you will get two versions of the app on the phone and it can help to have both named differently.